webapp/README.md

```diff
@@ -0,0 +1,24 @@
+# webapp
```

Comment:
Please link to this `webapp/README.md` from your `README.md` in your root directory. The `README.md` in your root directory is your landing page. Otherwise your open source contributors don't know what to do :shrug: